webapp/lib/main_ui.dart

```diff
@@ -43,6 +43,15 @@ class CodaUI {
     fbt.init();
     auth.init();
     snackbar.init();
+
+    window.onError.listen((e) {
+      if (e is ErrorEvent) {
+        var error = e.error;
+        var stacktraceOrFilename = error is Error ? error.stackTrace : e.filename;
+        log.severe('${e.message}: ${stacktraceOrFilename}');
+        snackbar.showSnackbar(e.message, snackbar.NotificatonType.error);
```

Comment:
I don't think we want to do this just yet - I think Firebase throws a bunch of exceptions if it loses network for a long time. For now, let's just send them to the log

Reply:
Good point, removed, thanks!